src/main/java/com/salesforce/comdagen/config/NavigationCatalogConfiguration.kt

```diff
@@ -16,6 +16,9 @@ data class NavigationCatalogConfiguration(
 
     val templateName: String = "catalogs.ftlx",
 
+    /** probability to assign a productsetitem to a productset in a navigationcatalog. Currently hidden. */
```

Comment:
What does "currently hidden" mean in this context?

Reply:
The configuration option can be used to change the behaviour but there is currently no set value in sites.yaml effectively hiding the option. As this seems to be more useful for specific use cases the option has been hidden. For instance productsets with a parameter of 'minSetProducts' could still be visible with a number of lower products because not all of them are assigned to the NavigationCatalog. To reflect expected behaviour this value is 1.0F by default.